### PR TITLE
Integrate exam attendance monitoring page

### PR DESCRIPTION
**Pull request by Jithmi004**
This UI has 3 states.

1. When data is loading

Screenshot (22)

2. When no data is found

Screenshot (21)

3. When data is found

Screenshot (20)

### CODE REVIEW

**Review by chathuranga-jayanath-99 — APPROVED** [review]

File: src/pages/exam_attendance_report.js [hunk 1]
```diff
@@ -1,51 +1,80 @@
...
 const Report = () => {
-  const navigate = useNavigate()
+  const navigate = useNavigate();
+  const { examid } = useParams();
+  const { authFetch } = useAppContext();
+  const [attendance, setAttendance] = useState([]);
+  const [exam, setExam] = useState({});
+  const [schedule, setSchedule] = useState();
+  const [isLoadingExam, setIsLoadingExam] = useState(true);
+  const [isLoadingAttendance, setIsLoadingAttendance] = useState(true);
```
Comment (thilinawaks1999) [inline src/pages/exam_attendance_report.js]: Can we change this initial state to false and set it to true at the begining of the function that fetching data and set to false after function complete.

File: src/pages/exam_attendance_report.js [hunk 2]
```diff
@@ -1,51 +1,80 @@
...
 
 const Report = () => {
-  const navigate = useNavigate()
+  const navigate = useNavigate();
+  const { examid } = useParams();
+  const { authFetch } = useAppContext();
+  const [attendance, setAttendance] = useState([]);
+  const [exam, setExam] = useState({});
+  const [schedule, setSchedule] = useState();
+  const [isLoadingExam, setIsLoadingExam] = useState(true);
```
Comment (thilinawaks1999) [inline src/pages/exam_attendance_report.js]: Lets do same thing for this also

**Review by thilinawaks1999 — CHANGES_REQUESTED** [review]

**Review by Malith-19 — APPROVED** [review]

File: src/pages/exam_attendance_report.js (hunk 1 above)
Comment (Jithmi004) [reply src/pages/exam_attendance_report.js]: Done

File: src/pages/exam_attendance_report.js (hunk 2 above)
Comment (Jithmi004) [reply src/pages/exam_attendance_report.js]: Done

**Review by thilinawaks1999 — APPROVED** [review]

**Pull request merged**